src/main/content/_assets/js/instance-common.js

```diff
@@ -112,6 +112,44 @@ function fetchGithubUserDetails(githubUsername){
         .catch(error => console.error(`Permission denied or user ${githubUsername} doesn’t exist`, error));
     }
 
+function addTeamMember(target) {
+    let teamId = $(target).closest(".bx--accordion__item").find(".admin-modal-accordion-title").attr("teamId");
+    let githubUsername = $(target).closest(".addUser-input-container").find(".add-user-text-input")[0].value;
```

Comment:
I would add another var like `let $addUserContainer = $(target).closest(".addUser-input-container")` and replace all isntances of `$(target).closest(".addUser-input-container")` -> `$addUserContainer` helps clean up a lot of text and prevent jQuery from having to parse the html a bunch of times